user/enterprise/build-images.md

```diff
@@ -32,24 +32,71 @@ For example, in order to install a particular Ruby version which is not
 available on the default `travis:ruby` image, and make it persistent,
 you can run:
 
-```    
+```
       docker -H tcp://0.0.0.0:4243 run -it --name travis_ruby travis:ruby su travis -l -c 'rvm install [version]'
       docker -H tcp://0.0.0.0:4243 commit travis_ruby travis:ruby
       docker -H tcp://0.0.0.0:4243 rm travis_ruby
 ```
 
 ## Enabling Docker Builds
 
+### Worker machine configuration
+
 In order to build other docker images, the Worker needs to be setup to support Docker Builds. To get started, please add the following to `/etc/default/travis-worker` on all your Workers:
 
 ```
     export TRAVIS_WORKER_DOCKER_PRIVILEGED="true"
-```       
+```
+
+#### Additional configuration for Trusty build images
+
+With Trusty build images a few additional steps are required. Since Docker-ce can't run on its own inside another Docker container, it'll connect to the Host's Docker daemon to execute the respective commands.
+
+> For non-AWS deployments, please make sure that the worker machine has a private IP address configured on one of its interfaces.
+
+On Amazon EC2 each machine has a private IP address by default. That address will be used to connect to the Docker daemon.
+
+To get the address, please run `ifconfig` in your terminal. For EC2 Virtual Machines, usually `eth0` interface is what we're looking for.
```

Comment:
> usually `eth0` interface is what we're looking for.

I'd try:

> usually `eth0` interface is needed

Reply:
that said, @plaindocs will probably tell me off for using the passive voice, so:
> usually the `eth0` interface is correct. 

is _probably_ be better